bundles/bundles.go

```diff
@@ -149,66 +160,87 @@ func NewBundleInstaller(bc BundleConfig) (BundleInstaller, error) {
 	return &LuetInstaller{}, nil
 }
 
-// BundleInstall installs a bundle from a luet repo or a container image.
-type ContainerRunner struct{}
+// OCIImageExtractor will extract an OCI image
+type OCIImageExtractor struct{}
```

Comment:
some of the code here seems similar to the one in https://github.com/kairos-io/kairos-agent/pull/20/files#diff-0fc4681264236aaf7a03bcdc3f6856266254a9aa788da892387b8a0a4ffe7a59, can we somehow have it in the sdk, and consume in the agent?

Reply:
I see, no, maybe open an issue? if you think it could be a good improvement

Reply:
yes, it may be good to have it together, maybe if we need we can just split it afterwards. Let me rework at least the extraciton so its more generica and can be used by the other callers